viv_utils/emulator_drivers.py

```diff
@@ -356,12 +359,8 @@ def handle_jmp(self, op, avoid_calls=False):
         # because there's an API found at this target address.
 
         if self._handle_hook():
-            # some hook handled the call,
-            # so make sure PC is at the next instruction
-            assert emu.getProgramCounter() == pc + len(op)
```

Comment:
these asserts were uncovering a bunch of issues, maybe instead of stopping the program we can raise them up the chain